SUPPORTED_PLATFORMS.md

```diff
@@ -11,7 +11,7 @@
 | z/OS | Tier 2 | >= V2R2 | Maintainers: @libuv/zos |
 | Linux with musl | Tier 2 | musl >= 1.0 | |
 | SmartOS | Tier 3 | >= 14.4 | |
-| Android | Tier 3 | NDK >= r15b | |
```

Comment:
r15b should still do. NDK is aligned with android sdk version. It has it's own version.

Reply:
SDK version: https://developer.android.com/studio/releases/platforms#7.0
NDK version: https://developer.android.com/ndk/downloads/revision_history

Reply:
Maybe add here we are now only supporting android 7.0 ?

Reply:
Good idea, done.